rust/src/dns/mod.rs

```diff
@@ -18,6 +18,12 @@
 use log::*;
 use conf;
 
+pub mod parser;
+pub use self::parser::*;
+
+pub mod dns;
+pub use self::dns::*;
```

Comment:
Just left overs from getting modules to work I think. I can remove them by doing "use dns::dns::..." in the modules that need rust/src/dns/dns.rs...